backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/CurrentEnvironment.java

```diff
@@ -0,0 +1,3 @@
+package de.bund.digitalservice.ris.caselaw.domain;
+
+public record CurrentEnvironment(String name) {}
```

Comment:
I am not sure if I fully understood where the enviornment comes from - but does it make sense to give CurrentEnvirionment an enum ("production" | "staging" | "uat") ?

Reply:
It comes from the spring boot config yaml files, so I felt like using an enum would make this more cumbersome and not actually safer, but I am happy to be challenged here :) basically now in `EnvironmentConfig` line 11 it gets read directly from the yaml file as a string. I guess we could cast it to an enum there, but not sure if it brings a huge benefit 🤔  what do you think?

Reply:
Ok great, got it, makes sense to just take the string here :)